src/Microsoft.SqlTools.ServiceLayer/SqlProjects/SqlProjectsService.cs

```diff
@@ -43,10 +43,14 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(CloseSqlProjectRequest.Type, HandleCloseSqlProjectRequest, isParallelProcessingSupported: true);
             serviceHost.SetRequestHandler(NewSqlProjectRequest.Type, HandleNewSqlProjectRequest, isParallelProcessingSupported: true);
 
-            // SQL object script calls
+            // SQL object script functions
             serviceHost.SetRequestHandler(AddSqlObjectScriptRequest.Type, HandleAddSqlObjectScriptRequest, isParallelProcessingSupported: false);
```

Comment:
@caohai do you recall why parallel processing not supported for sql project requests?

Reply:
@alanrenmsft I don't remember reviewing this file in my PR. I think this is a newly added service.

Reply:
Correct, it's new.  The underlying SqlProject and MSBuild construction objects aren't explicitly threadsafe (could potentially hit some funky behavior with multiple operations hitting it at the same time), so I turned this off for operations that reach down that low.